client/src/views/home/HomePageHeader.vue

```diff
@@ -104,13 +141,24 @@
           />
           {{ $msTranslate(backButtonTitle ?? 'HomePage.topbar.backToList') }}
         </ion-button>
+
+        <ion-menu-button
+          id="main-content"
+          class="menu-button"
+          v-if="windowWidth < 576"
```

Comment:
Better export this value (576 px) from a variable in `windowSize.ts` in megashark. It's better to centralize such info in case we modify it or need to use it elsewhere.